addons/web_editor/static/src/xml/editor.xml

```diff
@@ -90,10 +78,33 @@
             <input type="hidden" name="filters" t-att-value="widget.firstFilters.join('_')"/>
             <input t-if="widget.options.res_id" type="hidden" name="res_id" t-att-value="widget.options.res_id"/>
             <input t-if="widget.options.res_model" type="hidden" name="res_model" t-att-value="widget.options.res_model"/>
-            <div class="o_media_image_card card w-100">
-                <div class="card-body p-0">
+            <div class="card w-100">
+                <div class="card-body">
```

Comment:
I don't understand the point of using a `card` in this task.

Reply:
In order to have a darken background in all tabs. Was asked in specs. Do you have a better idea ?

Reply:
Card backgrounds are supposed to be white. If it is not the case, this is because there is some CSS for that particular modal. Removing that card stuff and applying that CSS to the `.tab-pane` elements should be enough (all of this will **remove** line of code to achieve the specs :wink:)

Reply:
@qsm-odoo Btw, this was already the case in video tab and icons tab. Should it be also removed ?

Reply:
If it is easy, yes, this is probably a bad idea there too.